spec/cart_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'cart'
+
+describe Cart do
+
+end
```

Comment:
But more seriously, `Cart` has two methods: `list`, which is responsible for `reading the list`, and `select_item`, which is responsible for `adding an item to the list`. I think the second method is unnecessary, as it just replicates the functionality of `<<`.

So you could do something like this:

```ruby
describe Cart do
  describe '#list' do
    it 'starts empty' do
      expect(Cart.new.list).to be_empty
    end

    it 'can have items added' do
      cart = Cart.new
      # not sure if I have this syntax right but it's something like this
      expect { cart.list << :item }.to change { cart.list.count }.by(1)
    end
  end
end
```